File: app/Models/Patient.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;


/**
 * The Patient class represents the patient model.
 */
class Patient extends Model
{
    protected $table = 'patient'; // Set the table name

    protected $primaryKey = 'idpatient'; // Set the primary key field

    // Define the fields that are fillable
    protected $fillable = [
        'atmung',
        'blutung',
        'radialispuls',
        'triagefarbe',
        'transport',
        'dringend',
        'kontaminiert',
        'name',
        'longitude_patient',
        'latitude_patient',
        'user_iduser', // Assuming there is a foreign key to the User model
    ];

    /**
     * Define a relationship with the User model.
     *
     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
     */
    public function user()
    {
        return $this->belongsTo(User::class, 'user_iduser', 'iduser');
    }


    /**
     * Define a one-to-one relationship with the QRCodePatient model (assuming you have one).
     *
     * @return \Illuminate\Database\Eloquent\Relations\HasOne
     */
    public function qrCodePatient()
    {
        return $this->hasOne(QRCodePatient::class, 'patient_idpatient', 'idpatient');
    }


    /**
     * Define a one-to-one relationship with the Body model.
     *
     * @return \Illuminate\Database\Eloquent\Relations\HasOne
     */
    public function body()
    {
        return $this->hasOne(Body::class, 'idpatient', 'idpatient');
    }
}

File: src/PLS.Api/Models/Entities/Patient.cs
namespace PLS.Api.Models.Entities;

public class Patient
{
    public int Id { get; set; }
    public string? Atmung { get; set; }
    public string? Blutung { get; set; }
    public string? Radialispuls { get; set; }
    public string? Triagefarbe { get; set; }
    public string? Transport { get; set; }
    public string? Dringend { get; set; }
    public string? Kontaminiert { get; set; }
    public string? Name { get; set; }
    public double? LongitudePatient { get; set; }
    public double? LatitudePatient { get; set; }
    public int? UserIdUser { get; set; }
    public DateTime? CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }

    public User? User { get; set; }
    public Body? Body { get; set; }
    public AmbulanzprotokollPage1? AmbulanzprotokollPage1 { get; set; }
    public QrCodePatient? QrCodePatient { get; set; }
}
